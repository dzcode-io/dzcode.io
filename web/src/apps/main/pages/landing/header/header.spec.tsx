import { Header } from "./index";
import { IntlProvider } from "react-intl";
import { MemoryRouter } from "react-router-dom";
import { render } from "@testing-library/react";

describe("header component", () => {
  test("should render header component", () => {
    const { container } = render(
      <MemoryRouter>
        <IntlProvider locale={"en"} defaultLocale="en">
          <Header />
        </IntlProvider>
      </MemoryRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
