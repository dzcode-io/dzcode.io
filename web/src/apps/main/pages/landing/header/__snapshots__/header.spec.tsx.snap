// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component should render header component 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-7"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-center"
      style="width: 100%; height: 100%;"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-7"
      >
        <div
          class="MuiBox-root MuiBox-root-8 makeStyles-left-2"
        >
          <h1
            class="MuiTypography-root makeStyles-title-3 MuiTypography-body1"
          >
            Algerian 
            <span
              class="makeStyles-greenHighlight-1"
            >
              Open Source
            </span>
             Community
          </h1>
          <p
            class="MuiTypography-root makeStyles-description-6 MuiTypography-body1 MuiTypography-colorTextPrimary"
          >
            Whether you're a maintainer or an open-source enthusiast DzCode helps you
             
            <a
              href="/Projects"
            >
              find
            </a>
            , 
            <a
              href="/Contribute"
            >
              contribute
            </a>
             
            and 
            <a
              href="/Projects"
            >
              list
            </a>
             open-source projects that solve Algerian problems.
          </p>
          <div
            class="MuiBox-root MuiBox-root-9"
          >
            <a
              class="makeStyles-callToAction-4"
              href="/Contribute"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-5 MuiButton-containedPrimary MuiButton-containedSizeLarge MuiButton-sizeLarge"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Make a Contribution
                </span>
              </button>
            </a>
            <a
              href="/FAQ"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-5 MuiButton-textPrimary MuiButton-textSizeLarge MuiButton-sizeLarge MuiButton-disableElevation"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label"
                >
                  Have a question ?
                </span>
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-5"
      />
    </div>
  </div>
</div>
`;
